Looks real, seems the get item (`[...]`) syntax is overloaded by `FieldColumn` which I guess `col` is.

```
ERROR: corehq.apps.app_manager.tests.test_suite:SuiteTest.test_report_module
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_suite.py", line 723, in test_report_module
    app.create_suite(),
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/models.py", line 5559, in create_suite
    return SuiteGenerator(self, build_profile_id).generate_suite()
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/suite_xml/generator.py", line 45, in generate_suite
    DetailContributor(self.suite, self.app, self.modules, self.build_profile_id),
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/suite_xml/generator.py", line 36, in _add_sections
    contributor.get_section_elements()
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/suite_xml/sections/details.py", line 56, in get_section_elements
    for detail_type, detail, enabled in module.get_details():
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/suite_xml/features/mobile_ucr.py", line 40, in get_details
    yield (_get_summary_detail_id(config), _get_summary_details(config, self.domain), True)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/suite_xml/features/mobile_ucr.py", line 208, in _get_summary_details
    details=[detail, _get_data_detail(config, domain)]
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/suite_xml/features/mobile_ucr.py", line 289, in _get_data_detail
    fields=[_column_to_field(c) for c in config.report(domain).report_columns]
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/suite_xml/features/mobile_ucr.py", line 278, in _column_to_field
    xpath=_get_xpath(column),
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/suite_xml/features/mobile_ucr.py", line 239, in _get_xpath
    transform = col.get('transform', {})
AttributeError: 'FieldColumn' object has no attribute 'get'
```

try catch would probably work fine

--- a/corehq/apps/app_manager/suite_xml/features/mobile_ucr.py
+++ b/corehq/apps/app_manager/suite_xml/features/mobile_ucr.py
@@ -236,7 +236,11 @@ def _get_data_detail(config, domain):
                     )
                 return word_eval
 
-            transform = col['transform']
+            try:
+                transform = col['transform']
+            except KeyError:
+                transform = {}
+
             if transform.get('type') == 'translation':
                 default_val = "column[@id='{column_id}']"
                 xpath_function = default_val

